views/index.ejs

```diff
@@ -59,17 +56,5 @@
       <script src="./statics/javascript/file_loader-debug.js"></script>
     <% } %>
```

Comment:
these "file_loader" files are probably useless now

Reply:
I tried to remove them but they are still in use indirectly to load language file. I'll try to clean that in another PR.